Content.Client/Arcade/UI/ParadiseArcadeBoundUserInterface.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Threading.Tasks;
+using Content.Shared.Arcade;
+using JetBrains.Annotations;
+using Robust.Client;
+using Robust.Shared.Timing;
+using Serilog;
```

Comment:
Wrong logger but not master so don't care.

Reply:
Oh yeah. That was very much just temporary. Then it became permanent